owncloudApp/src/main/res/layout/file_details_fragment.xml

```diff
@@ -16,192 +16,144 @@
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="12dp"
+            android:gravity="end"
+            android:text=""
+            android:textAppearance="@style/TextAppearance.MaterialComponents.Body1"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@id/fdModifiedLabel"
+            app:layout_constraintTop_toBottomOf="@id/fdSize"
+            tools:text="@tools:sample/lorem[6]" />
+
+        <TextView
+            android:id="@+id/fdProgressText"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_below="@+id/fdDetailsContainer"
-            android:layout_margin="@dimen/standard_margin"
-            android:orientation="vertical">
-
-            <TextView
-                android:id="@+id/fdProgressText"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:text="@string/downloader_download_in_progress_ticker"
-                />
-
-            <LinearLayout
-                android:id="@+id/fdProgressBlock"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                android:layout_marginBottom="12dp"
-                android:layout_marginTop="12dp"
-                android:gravity="center"
-                android:orientation="horizontal"
-                >
-
-                <ProgressBar
-                    android:id="@+id/fdProgressBar"
-                    style="?android:attr/progressBarStyleHorizontal"
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_weight="1"
-                    android:indeterminate="false"
-                    android:indeterminateOnly="false"
-                    />
-
-                <ImageButton
-                    android:id="@+id/fdCancelBtn"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_marginStart="12dp"
-                    android:background="@android:color/transparent"
-                    android:src="@drawable/ic_action_cancel_grey"
-                    />
-
-            </LinearLayout>
-
-        </LinearLayout>
-
-    </RelativeLayout>
+            android:layout_marginTop="@dimen/activity_vertical_margin"
+            android:text="@string/downloader_download_in_progress_ticker"
+            app:layout_constraintTop_toBottomOf="@id/fdModified" />
+
+        <ProgressBar
+            android:id="@+id/fdProgressBar"
+            style="?android:attr/progressBarStyleHorizontal"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="@dimen/activity_horizontal_margin"
```

Comment:
I see that we use different values in some views. In this XML I can see `@dimen/activity_horizontal_margin`, `12dp` and `@dimen/standard_margin` at the moment. I think we could unify them to `standard_margin`. What do you think?

Reply:
Sure!